lib/test-infrastructure/test-helper.ts

```diff
@@ -0,0 +1,47 @@
+/// <reference path='../../typings/tsd.d.ts' />
+
+import child = require('child_process');
+import Promise = require('bluebird');
+import MockWrapper = require('./mock-blpapi-wrapper');
+
+export enum ServerType {
+    HTTP,
+    SOCKETIO,
+    WS
+};
+
+export function startServer(instructions: MockWrapper.IInstruction,
+                            sType: ServerType = ServerType.HTTP): Promise<child.ChildProcess>
+{
+    // Set blpSession Instructions
+    process.env.WRAPPER_INSTRUCTIONS = JSON.stringify(instructions);
+    var server = child.spawn('node',
+                             ['./lib/test-infrastructure/test-index.js', '--port=3000'],
+                             { stdio: [null, null, null, 'ipc'] });
+/*
+    server.stdout.on('data', (data: any): void => {
+      console.log('stdout: ' + data);
+    });
+    server.stderr.on('data', (data: any): void => {
+      console.log('stderr: ' + data);
+    });
+    server.on('close', (data: any): void => {
+      console.log('child process exited with code ' + data);
+    });
+*/
```

Comment:
This is kind of useful for debugging purpose(ie, pipe the child process's stdout to the main process to figure out what's going on). Maybe add const to control whether to see it and set it if needed?